plugin/evm/vm_warp_test.go

```diff
@@ -656,7 +656,6 @@ func testReceiveWarpMessage(
 	ethBlock := block2.(*chain.BlockWrapper).Block.(*Block).ethBlock
 	rules := params.GetExtra(vm.chainConfig).GetAvalancheRules(ethBlock.Time())
 	headerPredicateResultsBytes := customheader.PredicateBytesFromExtra(rules, ethBlock.Extra())
-	require.NotEmpty(headerPredicateResultsBytes)
```

Comment:
not sure why this is removed?

Reply:
This was added in libevm (not present in master), seems a bit off to have that diff, maybe it was added to debug something back then? 🤔

Reply:
I think `ParseResults` will end up returning an error if it's empty.

Reply:
Alignment with `master` is not the correct litmus test here; correctness of the `libevm` branch is what we should be aiming for. This was added for a reason; if that reason is incorrect then yes, remove it, but otherwise it should be kept as a requirement.

Reply:
It has no reason to be there, and, yes, I agree, given we won't squash-merge libevm, aligning with master doesn't help much, except maybe to compare the libevm and master branches for the time being.